test/client/submit.ts

```diff
@@ -0,0 +1,108 @@
+import { assert } from 'chai'
+
+import { RippledError } from 'xrpl-local/common/errors'
+
+import { Wallet } from '../../src'
+import { Transaction } from '../../src/models/transactions'
+import rippled from '../fixtures/rippled'
+import { setupClient, teardownClient } from '../setupClient'
+import { assertRejects } from '../testUtils'
+
+describe('submit', function () {
+  describe('client.submitTransaction', function () {
```

Comment:
since they don't need any overlapping infrastructure, can you put the two methods in their own test files?